nix/default.nix

```diff
@@ -3,30 +3,31 @@
 , config ? { }
 , overlays ? [ ]
 , sourcesOverride ? { }
-, checkMaterialization ? false
-, enableHaskellProfiling ? false
-}:
-let
-  sources = import ./sources.nix { inherit pkgs; }
-    // sourcesOverride;
-  haskellNix = import sources."haskell.nix" {
+, sources ? import ./sources.nix { } // sourcesOverride
+, haskellNix ? import sources."haskell.nix" {
```

Comment:
Do we even want these defaults? Given that we should always call this via `default.nix`, this just means we duplicate the definitions.